.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,10 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Thanks, but we're not adding new features
```

Comment:
This tells users we're not accepting features before they click 'Feature request':

<img width="716" alt="Screenshot 2023-12-06 at 3 35 39 PM" src="https://github.com/studiopress/.github/assets/4063887/df76acf2-255d-4cd9-997b-8325fa34115b">